### PR TITLE
Build app log page

### LINKED ISSUE #101: Build app logs page

### PR DESCRIPTION
**Pull request by cjdenio**
Closes #101

### CODE REVIEW

File: pkg/api/apps/get.go [hunk 1]
```diff
@@ -17,7 +17,8 @@ func handleGETApp(c *gin.Context) {
 	}
 
 	var app db.App
-	result := db.DB.Joins("INNER JOIN team_users ON team_users.team_id = apps.id").
+	result := db.DB.Joins("JOIN teams ON teams.id = apps.team_id").
+		Joins("INNER JOIN team_users ON team_users.team_id = teams.id").
```
Comment (anirudhb) [inline pkg/api/apps/get.go]: Can't you just do `INNER JOIN team_users ON team_users.team_id = apps.team_id`

**Review by anirudhb — CHANGES_REQUESTED** [review]
lgtm

File: pkg/api/apps/get.go (hunk 1 above)
Comment (cjdenio) [reply pkg/api/apps/get.go]: You make an excellent point.

File: pkg/api/apps/get.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ func handleGETApp(c *gin.Context) {
 	}
 
 	var app db.App
-	result := db.DB.Joins("INNER JOIN team_users ON team_users.team_id = apps.id").
+	result := db.DB.Joins("INNER JOIN team_users ON apps.team_id = team_users.team_id").
```
Comment (anirudhb) [inline pkg/api/apps/get.go]: no it's the other way around

**Review by anirudhb — CHANGES_REQUESTED** [review]

File: pkg/api/apps/get.go (hunk 2 above)
Comment (anirudhb) [reply pkg/api/apps/get.go]: It should be `ON team_users.team_id = apps.team_id`

Comment (cjdenio) [reply pkg/api/apps/get.go]: It shouldn't matter wither way

Comment (anirudhb) [reply pkg/api/apps/get.go]: Style

**Review by anirudhb — APPROVED** [review]

**Pull request merged**